site/_posts/2017-10-30-crossword-lwimiq.html.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Crossword LWiMIQ"
+author: d-m-u
+date: 2017-10-30
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+## A Puzzle for your Perusal
+
+Any puzzles, correctly completed and returned to me before next week's blog, will enter you into a drawing for a prize!
```

Comment:
Perhaps include a way to contact you for those who may not know how, e.g. https://gitter.im/d-m-u (without having to give out your email address)